gravitee-apim-gateway/gravitee-apim-gateway-services/gravitee-apim-gateway-services-sync/src/main/java/io/gravitee/gateway/services/sync/synchronizer/OrganizationSynchronizer.java

```diff
@@ -114,11 +120,26 @@ private Flowable<String> deployOrganization(Flowable<io.gravitee.definition.mode
             .parallel(PARALLELISM)
             .runOn(Schedulers.from(executor))
             .doOnNext(
-                organization -> {
+                organizationToDeploy -> {
                     try {
+                        Organization organizationDeployed = organizationManager.getCurrentOrganization();
+                        if (
+                            organizationDeployed != null &&
```

Comment:
🤔  This duplicates with  
https://github.com/gravitee-io/gravitee-api-management/blob/3.15.x/gravitee-apim-gateway/gravitee-apim-gateway-platform/src/main/java/io/gravitee/gateway/platform/manager/impl/OrganizationManagerImpl.java#L52

but I maybe it would also have its place here to avoid unnecessary work before calling the register. (not sure 🤷‍♂️ )

Reply:
And if we remove it from there I don't really see how to add test